pkg/manager/leaderelection.go

```diff
@@ -13,13 +14,18 @@ func (k *KubeMacPoolManager) waitToStartLeading() error {
 }
 
 func (k *KubeMacPoolManager) markPodAsLeader() error {
```

Comment:
Can you change this function name to something like `labelLeaderElectedPod`?

Reply:
changed to `AddLeaderLabelToElectedPod`. is that ok?

Reply:
perfect